contentgrid-appserver-events/src/main/java/com/contentgrid/appserver/events/EventHandlerConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package com.contentgrid.appserver.events;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.boot.autoconfigure.amqp.RabbitProperties;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@RequiredArgsConstructor
+public class EventHandlerConfiguration {
+    @Bean
+    @ConfigurationProperties(prefix = "contentgrid")
+    ContentGridEventHandlerProperties contentgridEventHandlerProperties() {
+        return new ContentGridEventHandlerProperties();
+    }
+
+    @Bean
+    @ConditionalOnBean({RabbitTemplate.class, RabbitProperties.class})
+    @ConditionalOnProperty(value = {"spring.rabbitmq.host"})
```

Comment:
I'm not sure how I feel about this.
Spring has a default for rabbitmq host when creating rabbit template and so on.
So I don't think we should make the creation of the event handler dependent on that.

If we want to be able to disable this, maybe we should have a separate `contentgrid.events.rabbitmq.enabled=false` or something?